src/main/java/com/github/wovnio/wovnjava/Api.java

```diff
@@ -31,12 +33,23 @@
     private final RequestOptions requestOptions;
     private final ResponseHeaders responseHeaders;
     private final String responseEncoding = "UTF-8"; // always response is UTF8
+    private final int DYNAMIC_LOADED_PAGE_TTL_MILI = 20 * 60 * 1000;
+    private final Clock clock;
 
     Api(Settings settings, Headers headers, RequestOptions requestOptions, ResponseHeaders responseHeaders) {
         this.settings = settings;
         this.headers = headers;
         this.requestOptions = requestOptions;
         this.responseHeaders = responseHeaders;
+        this.clock = Clock.systemDefaultZone();
```

Comment:
tbh it's better to move these dependencies as high as up as you can go, so I would move it to the caller of `new Api()` which should be the "main" servlet function